From 041149c7090ad7414ef0321c33ec0919f6047f26 Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Wed, 6 Dec 2017 16:00:32 +0800
Subject: [PATCH] arm64: dts: rockchip: remove a stub pin node for
 rk3399-linux.dtsi

This stub pin node would prevent the whole pinctrl driver
being loading. Then any board files include the common
rk3399 linux would meet the power failure.

Change-Id: Ib223eb517c879b3819e9d8da4c0d5750886897c6
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
index 07a20146f2d3..32299b2f0844 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
@@ -198,12 +198,6 @@
 			rockchip,pins = <1 3 RK_FUNC_1 &pcfg_pull_none>;
 		};
 
-		isp_prelight: isp-prelight {
-			/* ISP_PRELIGHTTRIG */
-			/* used as pwrdn in excavator Evb_ExtBoard_MipiCam */
-			/* rockchip,pins = <1 4 RK_FUNC_1 &pcfg_pull_none>; */
-		};
-
 		isp_flash_trigger_as_gpio: isp-flash-trigger-as-gpio {
 			/* ISP_FLASHTRIGOU */
 			rockchip,pins = <0 17 RK_FUNC_GPIO &pcfg_pull_none>;
-- 
2.35.3

